website/tables/aws/aws_cloudwatch_metric_stats_custom.md

```diff
@@ -0,0 +1,21 @@
+# Table: aws_cloudwatch_metric_stats_custom
+
+This table shows data for Cloudwatch Metric Stats Custom.
+
+https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html
```

Comment:
Can we update both the `Title` and the `Description` field of this table so that the documentation makes it clear what this table does and how it works?

Reply:
Added a blurb, more explanation in configuration.md. Also since table_options is an experimental feature it doesn't have to be 100% bombproof documented, we can add a blog post for it later.